packages/integration-sdk-runtime/src/synchronization/__tests__/index.test.ts

```diff
@@ -504,5 +538,5 @@ function createTestContext() {
     name: 'test',
   });
 
-  return { apiClient, logger, integrationInstanceId: 'test-instance' };
+  return { apiClient, logger, integrationInstanceId: 'test-instance-id' };
```

Comment:
@ndowmon @austinkelleher @VDubber This helper function and the other helper function that's used in these unit tests - `generateSynchronizationJob` - were out of sync and returning different `integrationInstanceId`s.

I figure I'll leave my 2 cents here. I'm interested to see what other people think. I've been bitten in the past multiple times by helper functions in tests, similar to this instance. So I've started to move away from helpers in unit tests, or at least am much more conservative when I do use them, for two main reasons:
a) I think enumerating mock objects in the tests themselves adds visibility into exactly what and why that object looks the way it does for that particular test. Often times in unit tests, you're going to be altering a field or two of a mock object to cover a different code flow for a specific test.
b) I've found an over abundance of helpers can lead to this exact scenario - where they become disconnected and start returning different things for the same field, despite them should being the same. 

I've also seen arguments for helpers in unit tests. I'm not totally against them, I've just been more careful when making/using them. Just figured I'd throw this in and see what others' opinions are on the topic.